README.md

```diff
@@ -42,14 +42,17 @@ Parse code coverage report generated by build and post code coverage status comm
 
 ## Master Coverage from Sonar
 
-You have option to take master coverage (base coverage) from your Sonar instance instead of
-tracking it by Jenkins Job.
+You have the option to get the master coverage (base coverage) from your SonarQube instance. Otherwise the plugin will keep track of the master coverage in the project configuration file.
 
 * Goto ```Manage Jenkins```
-* Find section of plugin settings
-* Fill ```Sonar URL``` field
+* Find section ```Coverage status for GitHub Pull Requests```
+* Fill ```Sonar URL``` field - e.g. `http://sonar.mycompany.com`
 * Turn On ```Use Sonar for master coverage```
-* Next build for PR will use Sonar
+* The next pullrequest build will use SonarQube as the the source for master coverage
+
+The plugin will try to find the project in SonarQube based on the repository name. If more than one projects match, a waring is logged, and the first one will be used to get the coverage data. 
+
+This feature is still very basic. e.g. authorization against SonarQube is not yet implemented. Feel free to contribute.
```

Comment:
It will be good if you add that info to plugin configuration help section ```com/github/terma/jenkins/githubprcoveragestatus/Configuration/help-useSonarForMasterCoverage.html:1```